### ISSUE TITLE
Windows: `--npmPackages` does not report packages with names including a "/"

### ISSUE THREAD

**Issue by thedavidprice** [tabrindle/envinfo#160]
We have two dependencies in our root `package.json` (We are using yarn workspaces, which I know envinfo does not support `package.json` within the workspaces themselves):

```
// ./package.json
...
"devDependencies": {
    "@redwoodjs/core": "^0.4.0",
    "netlify-plugin-prisma-provider": "^0.1.0"
  },
```

**Mac command usage**
Running `yarn envinfo --npmPackages` outputs:
```shell
  npmPackages:
    @redwoodjs/core: ^0.4.0 => 0.4.0 
    netlify-plugin-prisma-provider: ^0.1.0 => 0.1.0 
```

**Windows command usage**
Using Powershell, running `yarn envinfo --npmPackages` outputs:
```shell
  npmPackages:
    netlify-plugin-prisma-provider: ^0.1.0 => 0.1.0 
```

I've made attempts to specify the identification of the @redwoodjs package, but the command returns empty. 

Any suggestions or workaround for Windows at this time?

**Comment by ryhinchey** [comment]
@thedavidprice thanks for reporting this. I'm working on a fix here https://github.com/tabrindle/envinfo/pull/161

**Comment by ryhinchey** [comment]
@thedavidprice just a heads up that this fix went out in envinfo `7.5.1`

**Comment by thedavidprice** [comment]
@ryhinchey Thanks again! I was watching the repo and already upgraded and merged to `master` 🚀
